.github/workflows/ci.yml

```diff
@@ -0,0 +1,177 @@
+name: CI
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    php-cs-fixer:
+        runs-on: ubuntu-latest
+        name: Coding Standards
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+                  tools: php-cs-fixer, cs2pr
+
+            - name: PHP Coding Standards Fixer
+              run: php-cs-fixer fix --dry-run --format checkstyle | cs2pr
+
+    phpstan:
+        runs-on: ubuntu-latest
+        name: Static Analysis
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+                  tools: phpstan, cs2pr
+
+            - name: Get composer cache directory
+              id: composercache
+              run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            - name: Cache dependencies
+              uses: actions/cache@v2
+              with:
+                  path: ${{ steps.composercache.outputs.dir }}
+                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
+                  restore-keys: ${{ runner.os }}-composer-
+
+            - name: Install dependencies
+              run: composer install --prefer-dist
+
+            - name: Install PHPUnit dependencies
+              run: vendor/bin/simple-phpunit --version
+
+            - name: PHPStan
+              run: phpstan analyse --no-progress --error-format=checkstyle | cs2pr
```

Comment:
phpstan has native support for Github Actions. So using `phpstan analyse --no-progress` would be better:

- you would still have annotations
- you would have the full phpstan report in a human readable way in the logs of the action, which is relevant when there is lots of reported issues (github actions are limited in the number of annotations they can report)